media-gfx/prusaslicer/prusaslicer-2.6.1.ebuild

```diff
@@ -56,9 +58,11 @@ PATCHES=(
 	"${FILESDIR}/${PN}-2.6.0-dont-force-link-to-wayland-and-x11.patch"
 )
 
-S="${WORKDIR}/${MY_PN}-version_${MY_PV}"
-
 src_prepare() {
+	if has_version ">=sci-libs/opencascade-7.8.0"; then
```

Comment:
Just a note: this is fine, but the only reason this is fine is because opencascade changes subslot between <7.8 and >=7.8.

Reply:
Can you explain why it wouldn't be fine otherwise?